test/BUILD

```diff
@@ -9,9 +9,17 @@ rule_test(
     rule = "@examples//hello_lib:hello_lib",
 )
 
+config_setting(
```

Comment:
IS there a reason you can't user `//rust/platforms:windows`?

Reply:
There is not, fixed. Thank you!